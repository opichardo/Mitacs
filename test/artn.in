&ARTN_PARAMETERS
  !! Units definition::(eV,Ang,ps)  
  engine_units     ='lammps/metal'
  verbose          = 1
  zseed            = 42

  !! ARTn mode
  ninit            = 0 
  lpush_final      = .true. 
  nsmooth          = 3 
  nnewchance       = 30
  forc_thr         = 0.01
  nperp_limitation = 4, 10, 15, 20, 25, 30, 35, -1 
  restart_freq     = 0

  !! parameters for the push
  push_step_size   = 0.40
  push_mode        = 'rad'
  push_dist_thr    = 3.5
  push_ids         = 955 

  !! lanczos parameters
  lanczos_disp     = 1.0D-2
  lanczos_min_size = 3
  lanczos_max_size = 50 
  
  !! eigenpush parms
  eigen_step_size  = 0.1
  push_over        = 6.0
/
