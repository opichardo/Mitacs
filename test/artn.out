     ARTn-plugin::output
     version: 1.0.0
     git branch: DEVEL   commit: 5d4db04a
     Launched on (dd.mm.yyyy): 24.6.2025 at: 13:56:14



